python_todo/views.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, jsonify, request, render_template
```

Comment:
REST should work like this:

#### `GET /`
Status: 200 (Success)
Body: All resources as JSON

#### `GET /:id`
Status: 200 (Success)
Body: The requested resource as JSON

#### `POST /`
Status: 201 (Created)
Body: The newly created resource as JSON

#### `DELETE /:id`
Status: 204 (No content)
Body: Empty

#### `PUT /:id`
Here you have two options:

Status: 200 (Success)
Body: The updated resource as JSON

Status: 204 (No content)
Body: Empty

Your `create_task`, `complete_task`, `delete_task` endpoints are atypical from what you'd see in most REST APIs. I do understand why you chose to implement it this way: it makes the client side wayyyyy easier, because you don't have to think about *how* to update the view. However, you're making your backend work a little bit harder, which is the opposite of what we want to be doing these days.